### ISSUE TITLE
F16v5 E-fuse state persistence?

### ISSUE THREAD

**Issue by joshwlewis** [Pixelcontroller/f16v5_issues#13]
Hey y'all, I've got a number of props on ports that use data from my F16V5 but use 12v power from a power supply in another box. I really don't want the power supply powering the controller to compete with the other power supply. With my F16V3, I'd just yank the fuses on port 1,3, and 15, and this worked fine. With the F16V5, I can toggle the fuses in the UI like this:

<img width="698" alt="image" src="https://github.com/user-attachments/assets/62927020-4be6-4a16-ad68-3bb63be2adfa" />

But, these are forgotten every time the power cycles. It's annoying to do this every day. Is there a way to save the fuse settings?

**Comment by keithsw1111** [comment]
At the moment there is no way to save it. In this case dont connect the lights to the +ve line ... just to the ground and data and take the power from your supply. Of course all the grounds between the supplies must also be connected together.

**Comment by dpitts** [comment]
HI

Not sure I follow. If you do not connect the power wire of the pixels to F16V5 that is same as removing fuses, I think.

Thank you,

David Pitts
***@***.***

From: Josh W Lewis ***@***.***>
Sent: Thursday, December 12, 2024 4:45 PM
To: Pixelcontroller/f16v5_issues ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Pixelcontroller/f16v5_issues] F16v5 E-fuse state persistence? (Issue #13)


Hey y'all, I've got a number of props on ports that use data from my F16V5 but use 12v power from a power supply in another box. I really don't want the power supply powering the controller to compete with the other power supply. With my F16V3, I'd just yank the fuses on port 1,3, and 15, and this worked fine. With the F16V5, I can toggle the fuses in the UI like this:
image.png (view on web)<https://github.com/user-attachments/assets/62927020-4be6-4a16-ad68-3bb63be2adfa>

But, these are forgotten every time the power cycles. It's annoying to do this every day. Is there a way to save the fuse settings?

**Comment by joshwlewis** [comment]
> If you do not connect the power wire of the pixels to F16V5 that is same as removing fuses, I think.

Yes, definitely true. I could disconnect the power lead at the controller, or remove the v+ pin from the extension cable. Those would both persist through restarts. But -- those are a little easier to forget next year. Will I remember that port 15 has no V+ next Halloween, or will I remember that this random extension cord has no V+? By comparison, I get pretty clear indication both in the UI and on the board when the fuse is disabled. It's also a little easier/faster to reverse.

No, this isn't a serious blocking issue or anything, more of a feature request. When I bought the F16V5 based on the description, I assumed the toggle-able fuses would be persisted for use cases like this. I'm a little disappointed that it doesn't. I think toggle e-fuse functionality isn't super useful without it. I'd guess it could be set in the pixel outputs screen if that's easier than the home screen.
